consensus/polybft/extra.go

```diff
@@ -398,7 +398,7 @@ func (s *Signature) VerifyCommittedFields(validators AccountSet, hash types.Hash
 		return err
 	}
 
-	if !aggs.VerifyAggregated(blsPublicKeys, hash[:]) {
+	if !aggs.VerifyAggregated(blsPublicKeys, hash[:], bls.DomainCheckpointManager) {
```

Comment:
Could we pass the domain to the `VerifyCommittedFields` function, so that the caller specifies it, and not hardcoding it like this? Plus the name `VerifyCommittedFields` is not exactly true, maybe rename it just to `Verify` or `VerifySignature`, or `VerifyAggregatedSignature`.

Reply:
Fixed https://github.com/0xPolygon/polygon-edge/pull/1184/commits/0683391e4b1b15ecfe0f2e80572d0444a795fbbe